index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./style.css" />
+  <title>Portfolio Website</title>
+</head>
+
+<body>
+  <div class="header">
+    <div class="placeHolder">
+      <h1 class="head">Welcome</h1>
+    </div>
+    <div class="nbtn1">
+
+      <div class="v1"></div>
+      <div class="v2"></div>
+      <div class="v3"></div>
+
+    </div>
+  </div>
+
+  <div class="container">
+
+
+
+    <!--Main part-->
+
+    <div class="Headline">
+      <div class="Primary">
+        <h3 style="color: #172B4D;">Hey there. I'm Faisal</h3>
+        <h3 style="color:#36B37E;">I'm a software developer</h3>
+      </div>
+      <br>
+      <div class="supporting_text">
+        <p> I can help you build a product , feature or website Look through some of my work and experience! If you like
+          what you see and have a project you need coded,
+          don’t hestiate to contact me.</p>
+      </div>
+      <div class="social_media_icons">
+        <img src="./icons/twitter.png" alt="">
+        <img src="./icons/Linkedin icon.png" alt="">
+        <img src="./icons/medium.png" alt="">
+        <img src="./icons/github.png" alt="">
+        <img src="./icons/angellist.png" alt="">
+      </div>
```

Comment:
- [ ] The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). * Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.